src/Annotator/EntityAnnotator.php

```diff
@@ -57,15 +57,11 @@ public function annotate($path) {
 		$propertyHintMap = $helper->buildEntityPropertyHintTypeMap($schema);
 		$propertyHintMap = $this->buildExtendedEntityPropertyHintTypeMap($schema, $helper) + $propertyHintMap;
 		$propertyHintMap += $this->buildVirtualPropertyHintTypeMap($content);
+		$propertyHintMap += $helper->buildEntityAssociationHintTypeMap($schema);
```

Comment:
Didnt  $helper->propertyHints($associationHintMap)) get lost now?
Please also keep the noise changes at a minimum for the surrounding code. "composer cs-fix" helps.

Reply:
Looking at the merge I would actually prefer the association ones before the virtual ones.
We need to change the merging op.

Reply:
Yup, the association hints are added to the `$propertyHintMap` instead before building the hints (Line 64).

Sorry about the noise, I forgot you use a different standard than everybody else :)

Reply:
$propertyHintMap += $this->buildVirtualPropertyHintTypeMap()
$propertyHintMap += $helper->buildEntityAssociationHintTypeMap()

So virtual before assoc, fair enough then :+1: